eng/Versions.props

```diff
@@ -117,10 +117,10 @@
   </PropertyGroup>
   <PropertyGroup>
     <MicrosoftNETTestSdkPackageVersion>17.6.0</MicrosoftNETTestSdkPackageVersion>
-    <XunitPackageVersion>2.6.6</XunitPackageVersion>
+    <XunitPackageVersion>2.7.1</XunitPackageVersion>
     <XunitRunnerVisualStudioPackageVersion>2.8.0</XunitRunnerVisualStudioPackageVersion>
-    <XunitExtensibilityExecutionPackageVersion>2.6.6</XunitExtensibilityExecutionPackageVersion>
-    <XunitAssertPackageVersion>2.6.6</XunitAssertPackageVersion>
+    <XunitExtensibilityExecutionPackageVersion>2.7.1</XunitExtensibilityExecutionPackageVersion>
+    <XunitAssertPackageVersion>2.7.1</XunitAssertPackageVersion>
```

Comment:
```
D:\a\_work\1\s\src\TestUtils\src\Microsoft.Maui.IntegrationTests\Microsoft.Maui.IntegrationTests.csproj : error NU1603: Warning As Error: Microsoft.Maui.IntegrationTests depends on xunit.runner.console (>= 2.7.1) but xunit.runner.console 2.7.1 was not found. An approximate best match of xunit.runner.console 2.8.0 was resolved.

/Users/builder/azdo/_work/1/s/src/Controls/tests/DeviceTests/Controls.DeviceTests.csproj : error NU1603: Warning As Error: Microsoft.Maui.TestUtils.DeviceTests.Runners depends on xunit.runner.utility (>= 2.7.1) but xunit.runner.utility 2.7.1 was not found. An approximate best match of xunit.runner.utility 2.8.0 was resolved.
```

Reply:
Yeah I tried promoting them to the nuget feeds which didn't seem to help, haven't had a chance to try again.